Workbooks/SapMonitor/OsMetrics/OsMetrics.workbook

```diff
@@ -0,0 +1,2137 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "loadType": "always",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "## Infrastructure and operating system metrics"
+            },
+            "name": "Heading text"
+          },
+          {
+            "type": 9,
+            "content": {
+              "version": "KqlParameterItem/1.0",
+              "parameters": [
+                {
+                  "id": "d247dd32-c3b4-4c2e-8da1-ae98f2d993da",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "provider_table_exists",
+                  "type": 1,
+                  "description": "Check if the provider table exists in the logs space",
+                  "query": "let hasNonEmptyTable = (T:string) \r\n{ \r\ntoscalar( union isfuzzy=true ( table(T) | count as Count ), (print Count=0) | summarize sum(Count) ) > 0\r\n};\r\nlet TableName = 'Prometheus_OSExporter_CL';\r\nprint present=iif(hasNonEmptyTable(TableName), \"1\", \"0\")",
+                  "isHiddenWhenLocked": true,
+                  "timeContext": {
+                    "durationMs": 86400000
+                  },
+                  "queryType": 0,
+                  "resourceType": "microsoft.operationalinsights/workspaces"
+                }
+              ],
+              "style": "pills",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "name": "parameters - 13"
+          },
+          {
+            "type": 1,
+            "content": {
+              "json": "Please enable OS Metrics provider in the Azure monitor for SAP solutions",
```

Comment:
Need to confirm the final provider name that will be available in the providers dropdown for customers and change the warning accordingly